src/store/configure.store.ts

```diff
@@ -433,6 +433,25 @@ const actions = {
       }
     }
   },
+  resetSettings({ commit }: Store<ConfigureState>) {
```

Comment:
Possibly easier to use the `defaultState` object instead of resetting each field individually.

```
const mutations = {
  resetSettings(state: ConfigureState) {
    state = { ...defaultState }
  },
};

const actions = {
  resetSettings({ commit }: Store<ConfigureState>) {
    commit('resetSettings')
  },
};
```